src/System.Private.CoreLib/src/System/Diagnostics/StackTrace.CoreRT.cs

```diff
@@ -101,6 +101,9 @@ internal string ToString(TraceFormat traceFormat)
                 frame.AppendToStackTrace(builder);
             }
 
+            if (traceFormat == TraceFormat.Normal)
+                builder.Length -= Environment.NewLine.Length;
```

Comment:
Is this going to crash when there are no `_stackFrames` ?

Reply:
Can this happen? There's a check `if (_stackFrames == null)` at the start of the method to bail. However if `_stackFrames.Length == 0` then this would crash.

Reply:
Better safe than sorry, i'll add a check